eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpClusteredVariantDeclusteredWriterTest.java

```diff
@@ -64,6 +80,11 @@ public void writeDeclusteredRs() {
                                                                               clusteredVariant);
         writer.write(Collections.singletonList(variant));
 
+        List<DbsnpClusteredVariantEntity> rsEntities = mongoTemplate.find
+                (new Query(), DbsnpClusteredVariantEntity.class, DBSNP_CLUSTERED_VARIANT_DECLUSTERED_COLLECTION_NAME);
+
+        assertEquals(1, rsEntities.size());
+        assertEquals(variant, rsEntities.get(0));
     }
```

Comment:
Sorry I didn't mention this in the previous review. Could you add a test to check insertions of multiple distinct entities, and another one with duplicates?